lib/engine/game/g_1846.rb

```diff
@@ -50,8 +50,8 @@ class G1846 < Base
 
       def init_companies(players)
         super + @players.size.times.map do |i|
-          name = (i + 1).to_s
-          Company.new(sym: name, name: name, value: 0, desc: "Choose this card if you don't want to purchase a company")
+          name = "Pass on buying a private company this round"
```

Comment:
change name to "Pass" since it's already in description.

use i as the sym in the constructor

Reply:
added a commit, i'm not super familiar w/ ruby string interpolation so please don't block on me struggling with it if there's a particular approach you prefer